ext/mysqli/mysqli.stub.php

```diff
@@ -76,7 +76,7 @@ public function dump_debug_info() {}
      * @return bool
      * @alias mysqli_debug
      */
-    public function debug(string $options) {}
+    public function debug(string $message) {}
```

Comment:
I'd rename this the other way around, i.e. `mysqli_debug($options)`.

Reply:
Done!